src/core/style/_elevation.scss

```diff
@@ -0,0 +1,186 @@
+/**
+ * A collection of mixins and CSS classes that can be used to apply elevation to a material
+ * element.
+ * See: https://www.google.com/design/spec/what-is-material/elevation-shadows.html
+ * Examples:
+ *
+ * ```scss
+ * .md-foo {
+ *   @include $md-elevation(2);
+ *
+ *   &:active {
+ *     @include $md-elevation(8);
+ *   }
+ * }
+ * ```
+ *
+ * ```html
+ * <div id="external-card" class="md-elevation-z2"><p>Some content</p></div>
+ * ```
+ *
+ * For an explanation of the design behind how elevation is implemented, see the design doc at
+ * https://goo.gl/Kq0k9Z.
+ */
+
+$_umbra-color: rgba(black, 0.2);
+$_penumbra-color: rgba(black, 0.14);
+$_ambient-color: rgba(black, 0.12);
+
+$_umbra-elevation-map: (
```

Comment:
Add comment that these values have been hand-tuned by Google's designers and can't quite be generated programmatically?